cmd/helm/lint.go

```diff
@@ -60,14 +61,21 @@ func lintCmd(cmd *cobra.Command, args []string) error {
 		return errLintNoChart
 	}
 
-	issues := lint.All(path)
+	linter := lint.All(path)
 
-	if len(issues) == 0 {
+	if len(linter.Messages) == 0 {
 		fmt.Println("Lint OK")
+		return nil
 	}
 
-	for _, i := range issues {
+	for _, i := range linter.Messages {
 		fmt.Printf("%s\n", i)
 	}
+
+	if linter.HighestSeverity == support.ErrorSev {
+		fmt.Println("Lint failed")
+		os.Exit(1)
```

Comment:
The thing I didn't like about returning an error is that Cobra outputs `Error: Lint failed` instead of just `Lint failed`. Is there anyway to stop it from adding that prefix?

Though the prefix is not a big deal, so we can just do that if you would prefer.